go-controller/pkg/ovn/egressip.go

```diff
@@ -2195,6 +2204,13 @@ func checkEgressNodesReachabilityIterate(oc *Controller) {
 }
 
 func (oc *Controller) isReachable(node *egressNode) bool {
+	if oc.eIPC.egressIPNodeHealthCheckPort == 0 {
+		return oc.isReachableViaDial(node)
```

Comment:
this is the legacy check? can we rename it to legacy or something. Dial doesn't mean anything